src/character_recognition.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+'''
+Predict characters with a sliding window of stepsize=1.
+'''
+
+import os
+import logging
+import sys
+
+import numpy as np
+import cv2
+from scipy.stats import norm
+from skimage import measure
+
+from feature_extraction import get_features_for_window
+from character_training import load_model
+import config
+
+
+def predict_wordfile(filename, model, dictionary):
+    '''
+    Load a file and predict the whole image as word
+    '''
+    raise ValueError('Not implemented...')
+
```

Comment:
Issue is that i need to generate a probability map an stuff like that for character recognition to run..